### PR TITLE
Retry Sumo logs

### PR DESCRIPTION
**Pull request by proteusvacuum**
FYI @amstone326 

https://manage.dimagi.com/default.asp?276824#1506288

Inspiration from: https://github.com/dimagi/commcare-hq/pull/20881

### CODE REVIEW

**Review by gbova — APPROVED** [review]

**Review by snopoke — APPROVED** [review]

File: corehq/ex-submodules/phonelog/tasks.py [hunk 1]
```diff
@@ -24,6 +27,11 @@ def purge_old_device_report_entries():
     UserEntry.objects.filter(server_date__lt=max_age).delete()
 
 
-@task(queue='sumologic_logs_queue')
-def send_device_logs_to_sumologic(domain, xform, url):
-    SumoLogicLog(domain, xform).send_data(url)
+@no_result_task(queue='sumologic_logs_queue', default_retry_delay=10 * 60, max_retries=3, bind=True)
+def send_device_log_to_sumologic(self, url, data, headers):
+    with Session() as s:
+        s.mount(url, HTTPAdapter(max_retries=5))
```
Comment (emord) [inline corehq/ex-submodules/phonelog/tasks.py]: Does this mean it will try 15 times? 3 celery task, 5 time each?

**Review by emord — APPROVED** [review]

File: corehq/ex-submodules/phonelog/tasks.py (hunk 1 above)
Comment (proteusvacuum) [reply corehq/ex-submodules/phonelog/tasks.py]: yes

**Comment by amstone326** [comment]
Woohoo, thank you @proteusvacuum!

**Comment by mkangia** [comment]
🏆

**Pull request merged**